omnigibson/prims/rigid_prim.py

```diff
@@ -82,20 +78,20 @@ def __init__(
         )
 
     def _post_load(self):
+        # Create the view
+        self._rigid_prim_view_direct = RigidPrimView(self._prim_path)
+
+        # Set it to be kinematic if necessary
+        if "kinematic_only" in self._load_config and self._load_config["kinematic_only"]:
```

Comment:
should we also enforce the inverse in an `else` clause?

Reply:
Good call. Done